app/page/template/warning.htm

```diff
@@ -75,28 +75,28 @@
 
       <!-- ko if: visibleWarning() === z.viewModel.WarningsViewModel.TYPE.REQUEST_NOTIFICATION -->
         <div class="warning-bar warning-bar-feature bg-theme">
-          <div class="warning-bar-message"
-               data-bind="l10n_html: {'id': z.string.warningPermissionRequestNotification, 'substitute': '<span class=\'warning-bar-icon icon-envelope\'></span>'}"></div>
-          <span class="warning-bar-close icon-close button-round button-round-dark"
-                data-bind="click: closeWarning"
-                data-uie-name="do-close-warning"></span>
+          <div class="warning-bar-message" data-bind="l10nSafeHtml: {'id': z.string.warningPermissionRequestNotification, 'substitutions': {'replaceDangerously': {'icon': '<span class=\'warning-bar-icon icon-envelope\'></span>'}}}"></div>
+          <span class="warning-bar-close icon-close button-round button-round-dark" data-bind="click: closeWarning" data-uie-name="do-close-warning"></span>
         </div>
       <!-- /ko -->
 
       <!-- ko if: visibleWarning() === z.viewModel.WarningsViewModel.TYPE.UNSUPPORTED_INCOMING_CALL -->
         <div class="warning-bar warning-bar-feature bg-theme">
           <!-- ko ifnot: z.util.Environment.browser.chrome -->
-            <div class="warning-bar-message"><span data-bind="l10n_text: {'id': z.string.warningCallUnsupportedIncoming, substitute: z.util.SanitizationUtil.escapeString(name())}"></span>&nbsp;<a class="warning-bar-link" rel="nofollow noopener noreferrer" target="_blank" data-bind="l10n_href: {support: z.config.SUPPORT.ID.CALLING}, l10n_text: z.string.warningLearnMore"></a></div>
+            <div class="warning-bar-message">
+              <span data-bind="l10n_text: {'id': z.string.warningCallUnsupportedIncoming, substitute: z.util.SanitizationUtil.escapeString(name())}"></span>&nbsp;<a class="warning-bar-link" rel="nofollow noopener noreferrer" target="_blank" data-bind="l10n_href: {support: z.config.SUPPORT.ID.CALLING}, l10n_text: z.string.warningLearnMore"></a>
```

Comment:
I believe you can even remove the `substitute: z.util.SanitizationUtil.escapeString(name())` here (since it's now the default behavior of the binding)